src/visualization.py

```diff
@@ -141,6 +141,8 @@ def senti_circleplot(senti_df, student_id):
             tooltip=[
                 alt.Tooltip(cts.SENTI, title="polarity"),
                 alt.Tooltip(student_id, title="author"),
+                alt.Tooltip("Positive words", title="Positive words"),
```

Comment:
I'm partly to blame here. `Positive words` is used in several places across multiple files. I would recommend storing this in the `constants.py` and just refer to that in these modules.